content/en/getting_started/serverless/_index.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Getting Started with Serverless Monitoring
+kind: documentation
+
+further_reading:
+    - link: '/agent/basic_agent_usage/'
+      tag: 'Documentation'
+      text: 'Basic Agent Usage'
+---
+
+## Overview
+
+_Serverless_ is a model where developers build and run applications and services using a cloud provider, rather than managing infrastructure themselves. Datadog [Serverless Monitoring][10] collects metrics, logs, and traces from your serverless infrastructure, enabling you to monitor your application's health and performance.
+
+{{< img src="getting_started/serverless/serverless_view.png" alt="Serverless Monitoring: Serverless View, an explorer page" style="width:80%;">}}
```

Comment:
Shall we use the sample app (instead of exposing org 2 data)?

Reply:
+1